build.gradle

```diff
@@ -7,54 +7,39 @@ buildscript {
     }
 }
 
+plugins {
+  id 'org.openjfx.javafxplugin' version '0.0.8'
+}
+
+javafx {
```

Comment:
I think that better to avoid using this plugin. You can look at another option in my this project:
https://github.com/JavaSaBr/kiosk/blob/master/build.gradle